.github/workflows/dockerhub-pr-with-parameters.yml

```diff
@@ -0,0 +1,100 @@
+on: issue_comment
+name: DockerHub (PR) with parameters
+
+jobs:
+  check_comments:
+    name: Check comments for /build_pr
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check for Command
+        id: command
+        uses: xt0rted/slash-command-action@v1
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          command: build_pr
+          reaction: "true"
+          reaction-type: "eyes"
+          allow-edits: "false"
+          permission-level: maintain
+      - name: Act on the command
+        run: echo "The command was '${{ steps.command.outputs.command-name }}' with arguments '${{ steps.command.outputs.command-arguments }}'"
+      # Build container
+      - uses: xt0rted/pull-request-comment-branch@v1
+        id: comment-branch
+        with:
+          repo_token: ${{ secrets.GITHUB_TOKEN }}
+      - uses: actions/checkout@v2
+        if: success()
+        with:
+          ref: ${{ steps.comment-branch.outputs.head_ref }}
+      - run: git rev-parse --abbrev-ref HEAD
```

Comment:
we can remove these lines here

```
      - run: git rev-parse --abbrev-ref HEAD
      - run: git rev-parse --verify HEAD
      - name: Clone branch of the PR
        uses: actions/checkout@v2
        with:
          token: ${{ secrets.GITHUB_TOKEN }}
          ref: ${{ steps.pr_data.outputs.branch }}
```
because we already cloned the branch in line 26